src/Monolog/Handler/RotatingFileHandler.php

```diff
@@ -166,7 +166,7 @@ protected function getGlobPattern()
         $fileInfo = pathinfo($this->filename);
         $glob = str_replace(
             ['{filename}', '{date}'],
-            [$fileInfo['filename'], '*'],
+            [$fileInfo['filename'], '[0-9][0-9][0-9][0-9]*'],
```

Comment:
This glob patter should as explicit as possible to cover the used file format. The current pattern covers only the year but not the month and day. This should be changed to something like this:

```php
            [$fileInfo['filename'], '20[0-9][0-9]-[01][0-9]-[0-3][0-9]*'],
```